app/controllers/api/service_dialogs_controller.rb

```diff
@@ -1,6 +1,13 @@
 module Api
   class ServiceDialogsController < BaseController
-    before_action :set_additional_attributes, :only => [:show]
+    def show
+      if params[:c_id]
+        resource = Dialog.find(params[:c_id])
+        render :json => single_resource(resource, params).target!
+      else
+        render :json => dialog_collection(params).target!
```

Comment:
@abellotti no they're not - these are IDs for dialog_group, dialog_tab, dialog_field. No APIs for them